rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/views/MonitoringSourceView.java

```diff
@@ -192,7 +192,11 @@ public void close() {
     @Override
     public void onFailure(Throwable caught) {
         close();
-        status.setContents(JSONUtils.getJsonErrorMessage(caught));
+        String errorMessage = JSONUtils.getJsonErrorMessage(caught);
+        if(errorMessage.contains("HTTP 500")){
+            errorMessage = "Monitoring is not available on this node source";
```

Comment:
I suppose monitoring cannot be disabled for a node source. In that case it is clearly an error. What about : Monitoring is not available on this node source because of an internal error. More information are available inside the scheduler logs.